packs/hubot/actions/post_result.py

```diff
@@ -2,7 +2,7 @@
 import httplib
 import requests
 import six
-import yaml
```

Comment:
I didn't think there was a yaml library shipped with python by default. This shouldn't have worked with pyyaml. It appears there is a pyyaml and a pyaml. pyaml looks like the one you want.